src/screens/Leaderboard/Leaderboard.tsx

```
@@ -173,115 +115,70 @@ function leaderboard(): JSX.Element {
-                    alt={firstName + ' ' + lastName}
-                  />
-                </div>
-              )}
-              {firstName + ' ' + lastName}
-            </div>
-          </>
+          <div
+            className={`${leaderboardStyles.volunteerCell}`}
+            onClick={() => navigate(`/member/${orgId}`, { state: { id: _id } })}
+            data-testid="userName"
+          >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)